source/install/desktop.rst

```diff
@@ -16,6 +16,10 @@ Below is a list of additional resources:
  - `Changelog <https://docs.mattermost.com/help/apps/desktop-changelog.html>`_
  - Contributor’s guide (coming soon)
  - `Source code <https://github.com/mattermost/desktop>`_
+ 
+.. note::
```

Comment:
Can you integrate this info with the troubleshooting section at the bottom of this doc? Maybe something like "If these troubleshooting tips are of no help, see..."